code/modules/surgery/bodyparts/head.dm

```diff
@@ -159,3 +160,7 @@
 
 	if(standing.len)
 		return standing
+
+/obj/item/bodypart/head/burn()
```

Comment:
No, it still drops the organs properly, the only thing missing was putting the brainmob into the brain beforehand.